salt/runners/cache.py

```diff
@@ -95,7 +76,7 @@ def pillar(tgt=None, tgt_type='glob', **kwargs):
     return cached_pillar
 
 
-def mine(tgt=None, tgt_type='glob', **kwargs):
+def mine(tgt=None, tgt_type='glob'):
```

Comment:
Are we quite certain nothing else is coming in through kwargs here?

Reply:
I was waffling on this one, because `kwargs` isn't used in this function, but after looking at the test results I think removing this is premature. I'll add it back in.